docs/source/references/api_spec.md

```diff
@@ -323,37 +347,34 @@ Learn [how to use metrics](https://hyperledger.github.io/iroha-2-docs/guide/adva
 **Expects**: -
 
 **Responses**:
+
 - 200 OK: The current version of API used by Iroha returned as a json string.
-Grabbed from the genesis block's version, so at least a minimal subnet of 4 peers
-should be running and the genesis be submitted at the time of request.
+  Grabbed from the genesis block's version, so at least a minimal subnet of 4 peers
+  should be running and the genesis be submitted at the time of request.
+
 ```
 "1"
 ```
 
-
 ## Parity Scale Codec
 
-For more information on codec check [Substrate Dev Hub](https://substrate.dev/docs/en/knowledgebase/advanced/codec) and codec's [Github repository](https://github.com/paritytech/parity-scale-codec).
+For more information on codec check [Substrate Dev Hub](https://substrate.dev/docs/en/knowledgebase/advanced/codec) and
+codec's [GitHub repository](https://github.com/paritytech/parity-scale-codec).
 
 ## Reference Iroha Client Implementation
 
 [Iroha client in Rust.](../../../client)
 
 ## Iroha Structures
```

Comment:
how is this section relevant?

Reply:
This section contains all the structures mentioned in the whole document. The purpose of it is to know the exact location of where structs are defined. 

At least it is my understanding of why this section was here. For me, as an SDK developer, it is useful, but I often found that the core team developers don't actualise this table.

Reply:
> core team developers don't actualise this table. 

Actually if we could just delegate it to automatically generated docs, we won't need to.

Reply:
I think it would be great if the whole `api_spec` is replaced with, for example, top-level documentation of the crate that is responsible for API. It will allow at least to use direct rustdoc links.

Reply:
So should we add the doc comments to `iroha_cli`? If this reduces the maintenance, and we only need to maintain the doc comments in one place, it'd save us a lot of headache.

Reply:
See #3296